dev-scripts/make-cli-executable.ts

```diff
@@ -0,0 +1,8 @@
+import { chmodSync } from 'fs';
+
+// See: https://github.com/microsoft/TypeScript/issues/37583
```

Comment:
The more commonly used permissions for executable files are `755` (the default is 644, not 664). The executable flag is added for others, group, and owner.

Reply:
Yeah, I used 755 before as well - but I decided to respect the defaults. Do we need 755 for some reason?

Reply:
But the default is `644`. Which means everybody can at least read the file. It makes sense that in case of a binary everybody would be able to run it as well. You can't know in what setup it's going to be used. Being able to execute the file for a group might be needed. I don't really see a reason not to add the execution flag for group and others as well.